RELEASE_NOTES.md

```diff
@@ -5,8 +5,10 @@
 ### New Features
 
 ### Bug Fixes
+- Fix OneHotEncoder Python serialization
```

Comment:
could i please ask you to move this to the breaking changes section, or mention it there as well? 

i think Talal mentioned that we should make a note that models serialized with older versions of mleap will not be deserialized correctly on upgrade, so i think we should mention that.

Reply:
Hi Anca, I don't think this is actually a breaking change...

The line that Talal was referring to was this one here:
https://github.com/combust/mleap/pull/664/files#diff-47b26af574b6dfeab149aeaa450acefa3ed9e37daa90407b90a8076ded3a5db4L654

Old bundles would still have the `{"size": n}` attribute serialized on disk. We are only changing what this gets deserialized into because the `n_values_` property has been deprecated.

Reply:
👍